modules/op-release-notes-1-19.adoc

```diff
@@ -168,15 +168,30 @@ apiVersion: operator.tekton.dev/v1alpha1
 kind: TektonConfig
 metadata:
   name: config
+spec:
   options:
     deployments:
-       tekton-results-api:
-         spec:
-           template:
-             spec:
-               containers:
-               - name: api
-                 env:
+      tekton-results-api:
+        spec:
+          template:
+            spec:
+              containers:
+                - name: api
+                  env:
+                    - name: LOGS_API
+                      value: "true"
+                    - name: LOGS_TYPE
+                      value: "Splunk"
+                    - name: LOGGING_PLUGIN_NAMESPACE_KEY
+                      value: "kubernetes.namespace_name"
+                    - name: LOGGING_PLUGIN_CONTAINER_KEY
+                      value: "kubernetes.container_name"
+                    - name: SPLUNK_SEARCH_TOKEN
+                      value: "Token that has permission to use the `Search` API in Splunk."
+                    - name: LOGGING_PLUGIN_API_URL
+                      value: "The URL of Splunk that includes the port."
+                    - name: LOGGING_PLUGIN_QUERY_PARAMS
+                      value: "Specify the index as `index=name`, where `name` is the index name."
```

Comment:
QE had not verified the issue. If SME had already verified, we can proceed with this.